string emotion
string image_1
string image_2
string image_x
string correct_image
